### PR TITLE
Add support for list user projects

### PR DESCRIPTION
**Pull request by ahhda**
Implements list user project part of https://github.com/PyGithub/PyGithub/issues/606#issuecomment-595864374

### Manual testing:
Screenshot 2020-03-10 at 1 52 53 AM

### CODE REVIEW

**Comment by s-t-e-v-e-n-k** [comment]
This looks great, you just need to fix the flake8 errors.

**Comment by ahhda** [comment]
Fixed the errors and rebased

File: github/NamedUser.py [hunk 1]
```diff
@@ -481,6 +483,21 @@ def get_orgs(self):
             github.Organization.Organization, self._requester, self.url + "/orgs", None
         )
 
+    def get_projects(self, state="open"):
+        """
+        :calls: `GET /users/:user/projects <https://developer.github.com/v3/projects/#list-user-projects>`_
+        :param state: string
+        :rtype: :class:`github.PaginatedList.PaginatedList` of :class:`github.Project.Project`
+        """
+        url_parameters = {"state": state}
```
Comment (s-t-e-v-e-n-k) [inline github/NamedUser.py]: Missing an assert that state is a str

Comment (ahhda) [reply github/NamedUser.py]: Resolved

**Pull request merged**